examples/tutorial/fortran/CMakeLists.txt

```diff
@@ -4,16 +4,35 @@
 #
 # SPDX-License-Identifier: (MIT)
 ##############################################################################
-blt_add_executable(
-  NAME tut_allocator_f
-  SOURCES tut_allocator.f
-  DEPENDS_ON umpire)
-list(APPEND umpire_tutorials tut_allocator_f)
+if(ENABLE_HIP)
```

Comment:
This can be tidied up. You can make an `tut_fortran_depends` variable and add `umpire`, then conditionally add `cuda_runtime` or `hip_runtime` based on which options are enabled. You can look at how the other tests are built for example.